NachoSpigot-Server/src/main/java/net/minecraft/server/ItemBoat.java

```diff
@@ -51,23 +51,24 @@ public ItemStack a(ItemStack itemstack, World world, EntityHuman entityhuman) {
             if (flag) {
                 return itemstack;
             } else {
-                if (movingobjectposition.type == MovingObjectPosition.EnumMovingObjectType.BLOCK) {
-                    BlockPosition blockposition = movingobjectposition.a();
+                if (movingobjectpositionblock.type == MovingObjectPosition.EnumMovingObjectType.BLOCK) {
 
+                    BlockPosition blockposition = movingobjectpositionblock.a();
+                    /* Paper - Remove unneeded interaction event trigger
```

Comment:
Very questionable removing this, would put this to a vote honestly

Reply:
I agree on this one. @crafter23456 could you for the time please revert this change?
You could put this in a new PR, and then we can vote there. Just so we don't block this PR.